app/components/community_activity_component/community_activity_component.html.erb

```diff
@@ -20,7 +20,7 @@
 
     <span class="community-activity-component__objective">
       <span class="community-activity-component__objective-text">
-        <span><%= @activity.title %></span>
+        <span><%= @activity.title.html_safe %></span>
```

Comment:
The document specified links in the title and evidence brief of some activities - not sure if this would cause any issues elsewhere?

Reply:
Changed this to santize to prevent undesired HTML elements